Sources/VaporToolbox/Run.swift

```diff
@@ -21,76 +20,111 @@ public final class Run: Command {
     }
 
     public func run(arguments: [String]) throws {
-        let folder: String
+        do {
+            let executable = try executablePath(arguments)
+
+            let configuredRunFlags = try Config.runFlags()
+            let passThrough = arguments + configuredRunFlags
+
+            let packageName = try extractPackageName()
+            console.info("Running \(packageName) ...")
+
+            try console.foregroundExecute(
+                program: executable,
+                arguments: passThrough
+            )
+        } catch ConsoleError.execute(_) {
+            throw ToolboxError.general("Run failed.")
+        }
+    }
 
+    private func buildFolder(_ arguments: [String]) throws -> String {
+        let folder: String
         if arguments.flag("release") {
-            folder = "release"
+            folder = ".build/release"
         } else {
-            folder = "debug"
+            folder = ".build/debug"
         }
 
         do {
-            _ = try console.backgroundExecute(program: "ls", arguments: [".build/\(folder)"])
+            _ = try console.backgroundExecute(program: "ls", arguments: [folder])
         } catch ConsoleError.backgroundExecute(_) {
-            throw ToolboxError.general("No .build/\(folder) folder found.")
+            throw ToolboxError.general("No \(folder) folder found.")
         }
 
-        do {
-            let name: String
-
-            if let n = arguments.options["name"]?.string {
-                name = n
-            } else if let n = try extractName() {
-                name = n
-            } else {
-                if arguments.options["name"]?.string == nil {
-                    console.info("Use --name to manually supply the package name.")
-                }
-
-                throw ToolboxError.general("Unable to determine package name.")
-            }
+        return folder
+    }
 
-            let exec = arguments.options["exec"]?.string ?? "App"
+    private func executablePath(_ arguments: [String]) throws -> String {
```

Comment:
95% of the time people will want to run `.build/debug/Run`

we should only ask them which executable to run if that one is not found.